src/Oscar.jl

```diff
@@ -249,6 +249,10 @@ function __init__()
 
     add_verbose_scope(:K3Auto)
     add_assert_scope(:K3Auto)
+    
+    add_verbose_scope(:GlobalTateModel)
+    add_verbose_scope(:GlobalWeierstrassModel)
```

Comment:
@thofma Initially, I tried to add these lines to `Oscar.jl/experimental/FTheoryTools/src/FTheoryTools.jl`. This seemed a natural place, given that `FTheoryTools` is experimental.

Still, if added to `Oscar.jl/experimental/FTheoryTools/src/FTheoryTools.jl` I received an `ErrorException` showing the error message "Not a valid symbol". Why is that?

Reply:
They need to be added very early in the loading of a package, so must go into `__init__`. If `FTheoryTools` would have its own module, you could put it in the `__ini__` function of that module, like here:

https://github.com/oscar-system/Oscar.jl/blob/915aeb9de5c3ffe97ffcbeca20b6fbca8e61d22c/experimental/GModule/Cohomology.jl#L9-L15

But just putting those in the `__init__` of Oscar itself is also fine.

Reply:
Thank you. That makes sense.